drafttopic/utilities/wikiprojects_common.py

```diff
@@ -0,0 +1,54 @@
+import os
+import json
+
+
```

Comment:
add `logger = logging.getLogger(__name__)` and don't have it passed into functions